packages/nox/test/epetra/1Dfem/1DfemPrePostOperator.C

```diff
@@ -57,7 +57,9 @@ UserPrePostOperator::UserPrePostOperator(const NOX::Utils& u) :
   numRunPreIterate(0),
   numRunPostIterate(0),
   numRunPreSolve(0),
-  numRunPostSolve(0)
+  numRunPostSolve(0),
```

Comment:
Did the compiler not warn about these being uninitialized?  I think it should have.

In any case, this might demonstrate why we need to be enabling OpenMP in our standard CI and auto PR builds (i.e. #2317).